packages/@o3r/schematics/src/utility/index.ts

```diff
@@ -19,3 +19,4 @@ export * from './routes';
 export * from './sub-entry';
 export * from './template-property.helper';
 export * from './update-imports';
+export * from './wrapper';
```

Comment:
not super explicite file name (but I do not have better :D)

Reply:
change to `metrics`